website/content/docs/concepts/policies.mdx

```diff
@@ -170,11 +170,30 @@ corresponding path and capability - even Vault's internal core configuration
 endpoints live under the `"sys/"` path. Policies define access to these paths and
 capabilities, which controls a token's access to credentials in Vault.
```

Comment:
I noticed the preview showed a line break after `"sys/"`, but I don't understand why.